<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>381</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>Основная информация</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="lblCode">
         <property name="text">
          <string>Код</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="edtCode"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblName">
         <property name="text">
          <string>Наименование</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QCheckBox" name="chkLaterality">
         <property name="text">
          <string>Латеральность</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="edtName"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="edtLatinName"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblLatinName">
         <property name="text">
          <string>Английское наименование</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblGroup">
         <property name="text">
          <string>Группа</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblArea">
         <property name="text">
          <string>Анатомическая область</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="CRBComboBox" name="cmbGroup"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblSynonyms">
         <property name="text">
          <string>Синонимы</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="edtArea"/>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="edtSynonyms"/>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabIdentification">
      <attribute name="title">
       <string>Идентификация</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CInDocTableView" name="tblIdentification"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CInDocTableView</class>
   <extends>QTableView</extends>
   <header location="global">library/InDocTable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
